app/javascript/packages/phone-input/index.ts

```diff
@@ -152,16 +176,14 @@ export class PhoneInputElement extends HTMLElement {
       });
 
       // Improve base accessibility of intl-tel-input
-      iti.flagsContainer.setAttribute('aria-label', this.strings.country_code_label);
-      iti.selectedFlag.setAttribute('aria-haspopup', 'true');
```

Comment:
As mentioned in the test inline code comment, a `combobox` role has a default `aria-haspopup="listbox"`, so this removes the assignment as redundant and less specific.

>Elements with the role combobox have an implicit [`aria-haspopup`](https://w3c.github.io/aria/#aria-haspopup) value of `listbox`.

https://w3c.github.io/aria/#combobox